payu/models/access.py

```diff
@@ -89,61 +100,68 @@ def setup(self):
             if model.model_type in ('cice', 'matm'):
 
```

Comment:
The previous calculations were identical for both `cice` and `matm`. It was simplest to make the same changes for both rather than keep all the old code for `matm` and add the new code separately for `cice`. This means that anyone who wants to run with `matm` would need to add the new restart file. Would this be ok to impose?

Reply:
I'm of the opinion we should drop support for `matm`.

I have made an issue to this effect

https://github.com/payu-org/payu/issues/486

But for the time being, I wouldn't worry about breaking `matm`. Or we can removed it and rebase this PR and it goes away.